src/main/java/core/basesyntax/FigureProducer.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureProducer {
+    public static final int COUNT_OF_FIGURE_IMPLEMENTATIONS = 5;
+    ColorProducer color = new ColorProducer();
```

Comment:
don't forget about access modifiers, should be default

Reply:
done